e2e/frontend/products.js

```diff
@@ -1,28 +1,36 @@
-module.exports = {
-    after: function (client) {
-        client.end();
-    },
+import { after, describe, it } from 'selenium-webdriver/testing';
+import driver from 'nodium/lib/driver';
+import chai, { expect } from 'chai';
+chai.use(require('chai-as-promised'));
+chai.use(require('chai-webdriver')(driver));
 
-    'Products - user should see the product list': function (client) {
-        client
-            .url('http://localhost:8081/#/products')
-            .waitForElementVisible('body', 1000)
-            .waitForElementVisible('.product-item', 1000, false, function () {
-                client.getLog('browser', function (result) {
-                    console.error(result);
-                });
+describe('Products', function () { // eslint-disable-line func-names
+    this.timeout(5000);
+
+    it('user should see the product list', done => {
+        driver
+            .get('http://localhost:8081/#/products')
+            .then(() => {
+                expect('.product-item').dom.to.have.count(3, done);
             });
-    },
+    });
 
-    'Products - user should see the product details': function (client) {
-        client
-            .url('http://localhost:8081/#/products/1')
-            .waitForElementVisible('.product-details', 5000);
+    it('user should see the product details', done => {
+        driver
+            .get('http://localhost:8081/#/products/1')
+            .then(() => {
```

Comment:
not a big fan of it, since nightwatch handles asynchronicity otherwise, why rely on promises?